### PR TITLE
Build Clean-Up and Reorg

### PR DESCRIPTION
**Pull request by CrackTheCode016**
This PR further organizes, removes, and cleans up the build section.

- Makes use of tabs to consolidate information
- Stick to more concise, up-to-date "one-pagers" rather than outdated guides
- Offer clear paths to external resources

### CODE REVIEW

**Comment by CrackTheCode016** [comment]
@DrW3RK @filippoweb3 I think I'll stop here with this PR - it really needs to be able to link out to something else, so I'll leave some current information until we have actual dev docs to link out to. 

I introduce a Build Archive, where pages that are mostly irrelevant live.  The rest of the pages left are pretty concise and should be up to date; let me know what you think!

File: docs/build/build-client-side.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+sidebar_label: Building Client-side Apps
+description: An overview about building clients that use Polkadot
+keywords: [data, index, query, explorer, dashboard, dapp, uapp, app, frontend, client]
+slug: ../build-client-side
+---
+
+import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
+
+As with any blockchain, building decentralized applications (dApps) is a significant part of how a
+developer can build on Polkadot. Within the ecosystem, you also hear the term "Unstoppable App", or
```
Comment (filippoweb3) [inline docs/build/build-client-side.md]: I would add conditional rendering for Kusama. I would remove the term "unstoppable" as not appropriate from the regulatory point of view. better using resilience

File: docs/build/build-client-side.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+uApp. While the naming isn't too important, what is important is building resilient applications
+using decentralized technology.
+
+As an application developer, you can compose your front-end apps in a few different ways. Because
+Polkadot and its parachains are all built using Substraste; you can often use the same SDK to
```
Comment (filippoweb3) [inline docs/build/build-client-side.md]:
```suggestion
Polkadot and its parachains are all built using the Polkadot SDK. You can often use the same SDK to
```

File: docs/build/build-client-side.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+Please see the [SS58 registry](https://github.com/paritytech/ss58-registry/) to see which
+[chain corresponds to a given prefix](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json),
+and which prefixes are available.
+
+:::
+
+## SDKS & Libraries
+
+If one aims to develop a **dApp** (Decentralized App) or **uApp** (Unstoppable App), the Polkadot
```
Comment (filippoweb3) [inline docs/build/build-client-side.md]: Same as before about "unstoppable"

File: docs/build/build-client-side.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+<TabItem value="pcloud">Polkadot Cloud hosts a library of assets, ranging from data sources,
+graphical elements, to fully functional components, for app developers to plug and play into their
+codebases. &nbsp;<a href="https://polkadot.cloud/" target="_blank">Learn more here</a>. </TabItem>
+
+</Tabs>
+
+## Oracle Options for Polkadot
+
+In the blockchain context, an _oracle_ is a way to bring real-world data onto the blockchain so that
```
Comment (filippoweb3) [inline docs/build/build-client-side.md:75]: there inbound and outbound oracles. Here you mention inbound only

File: docs/build/build-client-side.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+</Tabs>
+
+When using an oracle in your application you should be aware of the benefits and risks that are
+baked into its specific model.
+
+## Decentralized Storage Options
+
+Storage is an integral part of modern computer systems, and the same is true for distributed and
+decentralized systems like a blockchain. When interacting with the
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem, it will be helpful if you
```
Comment (filippoweb3) [inline docs/build/build-client-side.md:118]: I would make sure conditional rendering is applied to the whole page

**Review by filippoweb3 — CHANGES_REQUESTED** [review]
lgtm! It's a nice improvement to the previous situation. I added some comments and a few edits.

File: docs/build/build-client-side.md (hunk 4 above)
Comment (CrackTheCode016) [reply docs/build/build-client-side.md:75]: Do we have any good examples of outbound oracle solutions in polkadot atm? 

Isn't an outbound oracle just exporting various bits of data, which events kind of do already in substrate iirc

**Review by filippoweb3 — APPROVED** [review]

**Pull request merged**